store/tikv/lock_resolver.go

```diff
@@ -28,7 +28,8 @@ import (
 	goctx "golang.org/x/net/context"
 )
 
-const resolvedCacheSize = 512
+// ResolvedCacheSize max number of cached txn status.
+const ResolvedCacheSize = 512
```

Comment:
Is it necessary to make this variable public when no where else use it?

Reply:
I have move it to another pr #5788.